ecosystem/sep-0009.md

```diff
@@ -6,8 +6,8 @@ Title: Standard KYC / AML fields
 Author: stellar.org
 Status: Active
 Created: 2018-07-27
-Updated: 2020-04-03
-Version 1.2.0
+Updated: 2020-12-22
+Version 1.2.1
```

Comment:
As this change adds a new field, it should increment the minor number instead of the patch number.

```suggestion
Version 1.3.0
```